task/ecosystem-cert-preflight-checks/OWNERS

```diff
@@ -1,2 +1 @@
-
-opdev
+#opdev
```

Comment:
Could you explain what does it mean?

Reply:
I didn't know what it meant.. not assigned to any github id or team.. so commented it. Adam helped to find the person associated with this task. so changed it now.